open-api/swagger.yaml

```diff
@@ -753,6 +753,25 @@ paths:
           description: 'json that specifies the constraint. Example: `{"type":"study_name", "studyId":"EHR"}` or
           `{"type":"concept","path":"\\Public Studies\\CLINICAL_TRIAL_HIGHDIM\\High Dimensional data\\Expression Lung\\"}`.'
           type: string
+        - name: sort
+          required: false
+          in: query
+          type: string
+          description: >
```

Comment:
Did you mean: `description: |` ? 
The '>' symbol is breaking a formatting here

Reply:
What do you mean? This is valid yaml syntax, see e.g. https://yaml-multiline.info/. `|` preserves line breaks, `>` treats text as a wrapped line

Reply:
I am not sure if swagger interprets it correctly, the formatting of the generated documentation looks strange starting from this symbol (comparing to other parameters before this): 
![selection_073](https://user-images.githubusercontent.com/20908736/37760718-c64d48fc-2db7-11e8-863c-c6695e3f9726.png)

Reply:
Hmm that's strange. But I thought swagger used the swagger.json or swagger_spec.js files. There the syntax is just a normal javascript string.

Reply:
The difference seems to be due to required vs optional parameters. The `type` and `constraint` parameters have an extra `required` css class. So looks like it works as intended.

Reply:
ah, you are right! sorry!